fundingmanager.go

```diff
@@ -348,9 +359,8 @@ func (f *fundingManager) handleFundingRequest(fmsg *fundingRequestMsg) {
 	delay := msg.CsvDelay
 
 	// TODO(roasbeef): error if funding flow already ongoing
-	fndgLog.Infof("Recv'd fundingRequest(amt=%v, push=%v, delay=%v, pendingId=%v) "+
-		"from peerID(%v)", amt, msg.PushSatoshis, delay, msg.ChannelID,
-		fmsg.peer.id)
+	fndgLog.Infof("Recv'd fundingRequest(amt=%v, delay=%v, pendingId=%v) "+
+		"from peer(%v)", amt, msg.PushSatoshis, delay, msg.ChannelID, fmsg.peerAddress.IdentityKey)
```

Comment:
The last portion of this logging message should instead be `peer(%v)` and the compressed serialized version of the peer's public key should be passed.